app/src/main/java/io/gnosis/safe/ui/transaction/TransactionViewHolderFactory.kt

```diff
@@ -129,3 +285,18 @@ class SectionHeaderViewHolder(private val viewBinding: ItemTxSectionHeaderBindin
         }
     }
 }
+
+private fun statusTextColor(status: TransactionStatus, resources: Resources): Int {
+    return when (status) {
+        TransactionStatus.Success -> resources.getColor(R.color.safe_green)
+        TransactionStatus.Cancelled -> resources.getColor(R.color.dark_grey)
+        else -> resources.getColor(R.color.safe_failed_red)
+    }
+}
+
+private fun formatAmount(viewTransfer: Transaction.Transfer, incoming: Boolean): String {
+    val inOut = if (incoming) "+" else "-"
+    val symbol = viewTransfer.tokenInfo?.symbol
+    val value: String = viewTransfer.tokenInfo?.decimals?.let { viewTransfer.value.shiftedString(decimals = it) }.toString()
+    return "%s%s %s".format(inOut, value, symbol)
```

Comment:
seems here that value could be the string "null" ? Maybe it makes more sense to return an empty string if `value` is null?